app/templates/components/modals/tax-info-modal.hbs

```diff
@@ -1,3 +1,4 @@
+<i class="black close icon"></i>
```

Comment:
There is already a closable icon in the modal I believe?

Reply:
@iamareebjamal not present earlier. please see #4505